pontos/nvd/models/cve_change.py

```diff
@@ -0,0 +1,46 @@
+# SPDX-FileCopyrightText: 2023 Greenbone AG
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from dataclasses import dataclass
+from datetime import datetime
+from enum import Enum
+from typing import List, Optional
+from uuid import UUID
+
+from pontos.models import Model
+
+
+class EventName(str, Enum):
+    INITAL_ANALYSIS = "Initial Analysis"
+    REANALYSIS = "Reanalysis"
+    CVE_MODIFIED = "CVE Modified"
+    MODIFIED_ANALYSIS = "Modified Analysis"
+    CVE_TRANSLATED = "CVE Translated"
+    VENDOR_COMMENT = "Vendor Comment"
+    CVE_SOURCE_UPDATE = "CVE Source Update"
+    CPE_DEPRECATION_REMAP = "CPE Deprecation Remap"
+    CWE_REMAP = "CWE Remap"
+    CVE_REJECTED = "CVE Rejected"
+    CVE_UNREJECT = "CVE Unreject"
+
+    def __str__(self) -> str:
+        return self.value
+
+
+@dataclass
+class Detail:
+    type: str
+    action: Optional[str] = None
+    old_value: Optional[str] = None
+    new_value: Optional[str] = None
+
+
+@dataclass
+class CVEChange(Model):
+    cve_id: str
+    event_name: EventName
+    cve_change_id: UUID
+    source_identifier: str
+    created: Optional[datetime] = None
+    details: Optional[List[Detail]] = None
```

Comment:
Forgot to mention, with requiring Python >= 3.9 we can use [PEP 585](https://peps.python.org/pep-0585/) 
```suggestion
    details: Optional[list[Detail]] = None
```

Reply:
Done (+ a few other cases) in https://github.com/greenbone/pontos/pull/920/commits/40903811185bac08ce7fcf8f383e6eafadec0409.